packages/react-core/src/components/Dropdown/DropdownToggle.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Dropdown/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  isCollapsed: PropTypes.bool,
+  isKebab: PropTypes.bool,
+  ariaLabel: PropTypes.string
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  isCollapsed: true,
+  isKebab: false,
+  ariaLabel: 'Actions'
+};
+
+const DropdownToggle = ({
+  className,
+  children,
+  isKebab,
+  isCollapsed,
+  ariaLabel,
+  ...props
+}) => {
+  const additionalProps = props;
+  if (isKebab) {
```

Comment:
same comment lets let the user pass in what they want instead of locking to kebab or text.